src/connectedmachine/azext_connectedmachine/generated/action.py

```diff
@@ -31,7 +31,7 @@ def get_action(self, values, option_string):
                 properties[k].append(v)
             properties = dict(properties)
         except ValueError:
-            raise InvalidArgumentValueError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
```

Comment:
We recommend using more specific error type instead of `CLIError`, so please do not make these updates